pages/main.html

```diff
@@ -1,70 +1,54 @@
 <main class="flex flex-col justify-between h-screen py-2" id="main">
```

Comment:
Ya' know, I really tried to get a pretty looking git diff but I think someone must have special tab/space rules that none of my editors have. Haha oops! 😅 a default formatter and format config file would help with that, it's super helpful on open source projects.

Reply:
good thinking! go ahead and add one and if anyone has issues with it we can work out specific formatting ideas.

Reply:
Does everyone have Prettier or use an editor that could get prettier? I know VSCode has it

Reply:
Ya I think that should be fine, you could make the changes. 

Concerning the merge, Nikhil recently made it so once this is approved it should be merged automatically, you could optionally remove him as an approver, or nudge him to approve. I mean I could manually merge it, but that would circumvent the auto merge thing, which then would defeat the purpose of having it in place.

tldr, hopefully nikhil pops in.